source/includes/_users.md

```diff
@@ -260,6 +314,8 @@ name | string | The User's name.
 email | string | The User's corporate email.
 phone | string | The User's phone.
 password | string | The User's password.
+is_admin | bolean | The User's is_admin.
+role_category_ids | array | The User's role category ids.
```

Comment:
Specifies the categories the user is associated with. This is required in case your user isn't an administrator.

Reply:
done